runtime/jcl/common/stdinit.c

```diff
@@ -169,6 +173,12 @@ restoreRunStandardInit(J9JavaVM *vm)
 #endif /* !J9VM_INTERP_MINIMAL_JCL */
 #endif /* J9VM_OPT_SIDECAR */
 
+#if JAVA_SPEC_VERSION >= 11
+	vmFuncs->internalAcquireVMAccess(vmThread);
+	vmFuncs->sendInitEncodings(vmThread);
```

Comment:
Why is this required?  It inits the static fields {`platformEncoding`, `fileEncoding`, `osEncoding`} in System.java.  Aren't they saved in an initialized state?  Why do we reset them on a restore run?

Reply:
The JCL natives set a native static when encodings aare initialized, this needs to be re-init in the restore run

Reply:
Thanks.  Following that path, it looks like this is the native that needs to be called again:
https://github.com/eclipse/openj9/blob/0aec295571c6c25927af41a629bc915c2129642e/runtime/jcl/common/system.c#L425-L429

Reply:
Can you add this as a comment to the code?  Something like:
```
/* Need to resend System.initEncodings() because it calls into native code which
 * has to reinitialize the native static value done by the "InitializeEncoding" native.
 * See the code in system.c:: getEncoding
 */